### DISCUSSION TITLE
Can someone tell me why am I unable to see the GitHub details?

### DISCUSSION THREAD

**Discussion by rajtilakjee** [squidfunk/mkdocs-material#5594] [Q&A] [answered]
The GitHub details like the latest release, forks, and stars that are supposed to be shown on the header are not viewable. It was fine in local, and was fine a while back on GitHub pages. Here's the link to my site: https://rajtilakjee.github.io/SignLink/

**Comment by squidfunk** [comment, marked as answer]
Works for me:

<img width="232" alt="Bildschirm­foto 2023-06-08 um 19 57 46" src="https://github.com/squidfunk/mkdocs-material/assets/932156/30ca4fe0-afa0-4f4d-a84e-e2200251facb">

Make sure to clear your browser cache and that the requests against the GitHub API dont' 429 (too many requests)

**Reply by rajtilakjee** [reply]
Yes, it is working now. Thank you for the reply and for creating such a beautiful template.
